### ISSUE TITLE
export to mbox fails : creates a 0kb file with no errors

### ISSUE THREAD

**Issue by rzmu** [thunderbird/import-export-tools-ng#547]
windows 11
tools 14.0.01
thunderbird 115.9.0

I've exported a few office 365 folders to mbox without issue (sent items)
Now I'm trying to export the Inbox, first the entire thing (approx 10k messages), then just a smaller selection (moved about 2k to a subfolder first).

These exports just generate a 0kb file, without any information about what the problem could be.

**Comment by cleidigh** [comment]
@rzmu 
I owe you an answer from yesterday, but I will respond to this first.
Can you capture and post the console output (control-shift-j)
When you say 365, were these folders imported?
@cleidigh

**Comment by rzmu** [comment]
Thanks for the reply ! :-)
I am indeed less interested in the export to .eml issue I posted yesterday since exporting to .mbox is a more efficacious workflow.
[console-export-2024-3-21_22-19-11.txt](https://github.com/thunderbird/import-export-tools-ng/files/14707675/console-export-2024-3-21_22-19-11.txt)

Please find my console log attached.
I seem to be managing to export a smaller subfolder I made.
But trying to export the full inbox, which I would ideally like to do, gives me the same 0kb file.

Regarding the Inbox folder that I would like to export : no, it is not an imported folder. It is just the original Inbox in an office 365 account.

**Comment by cleidigh** [comment]
@rzmu 
ok you can send to my test account 
test1@kokkini.net

I have seen quirks with MS messages so I am expecting something there.
@cleidigh

**Comment by cleidigh** [comment]
got it...

**Comment by cleidigh** [comment]
@rzmu 
There are a lot of debug and also error messages from one or more extensions. One might be Owl?
I honestly don't know how those are related, but maybe disabling all other extensions is a start. If you copied the inbox to LocalFolders and did the export there that would eliminate the server.
@cleidigh

**Comment by rzmu** [comment]
Yes Owl is essential, it is the extension that enables one to connect to an office 365 account with thunderbird.
I will try copying to LocalFolders.

**Comment by rzmu** [comment]
No luck copying to LocalFolders so far, I get Mime Content Conversion Failed.

**Comment by cleidigh** [comment]
@rzmu 
There are a lot of OWL errors including mime conversion.
This makes me think message retrieval is having problems.
I wonder if that needs to be looked at by OWL
@cleidigh

**Comment by rzmu** [comment]
I'm pretty sure getting Thunderbird to sync properly to outlook 365 is pretty thorny ;-)
As far as I can tell, Owl is managing to get the messages from the server properly, and make them accessible in Thunderbird. (the numbers of messages match up, and the attachments all work).
Beyond that I don't know...

**Comment by cleidigh** [comment]
> No luck copying to LocalFolders so far, I get Mime Content Conversion Failed.

See that operation is TB+owl.
There is clearly an account server issue with owl.
This is also causing the eml issue 
@cleidigh

**Comment by rzmu** [comment]
If there is a particular message that importexporttools doesn't like, is there a log somewhere where I can find it ?

**Comment by cleidigh** [comment]
@rzmu 
How do mean IETNG does not like it? Are you getting errors on one specific message?
Can you copy that message to local folders?
@cleidigh

**Comment by rzmu** [comment]
I'm not getting errors on one specific message, but I get the feeling, since on many folders IETNG mbox export works, that one message might be the problem.

It's either that, or the size of the folder, I guess.

**Comment by cleidigh** [comment]
@rzmu 
The 10k is nothing.
I suspect multiple messages and or the server communication.
If you take IETNG out of the equation I expect you will still see a variety of problems.
I really think the owl folks need to look at those.
Unfortunately I have zero experience with owl
@cleidigh

**Comment by cleidigh** [comment]
@rzmu 
The view message source is one example. If you see a bunch of undecoded data its not detecting boundaries or failing to convert a content section.
@cleidigh
